### ISSUE TITLE
[Feature Request]: Restore Multi-inputs for Instant-LoRA workflows

### ISSUE THREAD

**Issue by strawberrymelonpanda** [lllyasviel/stable-diffusion-webui-forge#98]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What would your feature do ?

From [Discussions](https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/40#discussion-6184773)

> Multi-inputs is removed. You now need to enable multiple IP-Adapter units to achieve effect of previous multi-inputs
> We will add this back if there is a strong demand on that.

Yes, please! I was so stoked to see it added, and bummed to see it's now gone again. It's super important for IP Adapter "Instant LoRA" type workflows, and a whole lot faster than adding 2-4 different duplicate ControlNets to do the same thing.

### Proposed workflow

Restore the "Multi-Inputs" functionality.

**Comment by Zibbibbo** [comment]
Is there any method to manually re-activate it?

**Comment by strawberrymelonpanda** [comment]
No, it was removed.  You'd have to look at the multi-input commits from the previous extension, and then rework the code to this codebase. The workaround for now is to make multiple controlnets, and put one image in each.  It's just rather slow and cumbersome by comparison, setting up 2 or 3 or 4 duplicate controlnets.

Since @huchenlei has self-assigned this, fingers crossed he's on it. 🎉👍   _(thanks!)_
